ttg/ttg/tt.h

```diff
@@ -11,50 +11,65 @@
 
 namespace ttg {
 
+  /// @brief a template task graph implementation
+
```

Comment:
Can you add a sentence on how a TT is different from a TTG? The CompositeOp kind of made clear that it is a composite of Ops but I'm not sure that's clear anymore.